salt/kubernetes-minion/kubeconfig.jinja

```diff
@@ -1,13 +1,12 @@
-{% set kube_servers = [] -%}
-{% for fqdn in salt['mine.get']('roles:kube-master', 'fqdn', expr_form='grain').values() -%}
-  {% do kube_servers.append(fqdn) -%}
-{% endfor -%}
+{% set api_server = "api." + salt['pillar.get']('internal_infra_domain','')  -%}
```

Comment:
In HP Public Cloud many years ago we wrote a macro that did pillar value fallback. I can see if I can dig it out